api/cases/sscsCase.template.js

```diff
@@ -115,6 +115,20 @@ module.exports = {
                     ]
                 }
             ]
+        },
+        {
+            id: 'timeline',
+            name: 'Timeline',
+            type: 'page',
+            sections: [
+                {
+                    id: 'events',
+                    name: '',
+                    type: 'timeline-panel',
+                    fields: [
```

Comment:
Is it possible for us to have one api call to ccd rather than two ... meaning the events data is returned in /cases ?

 
please could we explicitly define the value required to be looked up in the response and assign it to fields array.

Reply:
Please could we add tests to the api to ensure the work around timeline is as expected.

Reply:
The direct call to get the data from CCD does not include the events. Only the aggregate call includes the events. We need to make a separate call to get the events data.

Reply:
As we cant QA this - I am happy for us to have 2 api calls to ccd for now.